.github/workflows/build.yml

```diff
@@ -255,20 +288,21 @@ jobs:
             docker.io/hashicorppreview/${{ env.repo }}-control-plane:${{ env.version }}-${{ github.sha }}
 
   build-docker-ubi-redhat-registry:
-    name: Docker ${{ matrix.arch }} UBI build for RedHat Registry
+    name: Docker ${{ matrix.arch }} ${{ matrix.fips }} UBI build for RedHat Registry
     needs: [get-product-version, build]
     runs-on: ubuntu-latest
     strategy:
       matrix:
         arch: ["amd64"]
+        fips: [ ".fips1402", "" ]
```

Comment:
This is the same as:

> { arch: "amd64", fips: "" }
> { arch: "amd64", fips: ".fips1402" }

isn't it?  If so, even if it is tedious, maybe make it the same format for future us?

Reply:
done